src/routes/auth.py

```diff
@@ -288,4 +288,49 @@ async def revoke_session(
             
             return RevokeResponse(revoked=True)
     except Exception as e:
-        raise_http_error(500, "Revocation failed")
\ No newline at end of file
+        raise_http_error(500, "Revocation failed")
+
+
+
+
+@router.post("/reset", response_model=SessionResponse)
+async def reset_session(
```

Comment:
endpoint for this purpose already exists. that is refresh-session

if frontend hasn't implemented refresh-session endpoint, then remove refresh-session, else migrate reset-session logic to refresh-session


without resolving this, this PR cannot be merged.